scripts/generate_driver_wrappers.py

```diff
@@ -7,7 +7,19 @@
    based on template files in script/data_files/driver_templates/.
 """
 # Copyright The Mbed TLS Contributors
-# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
oups, no.

Reply:
Oh whoops! That was because I was playing with the files locally for testing and must have used an older version of the repo for this, and then I just copied the scripts over once they worked. I will make sure to keep an eye out for this.